packages/backend/src/apps/slack/actions/find-message/index.ts

```diff
@@ -25,6 +25,7 @@ const action: IRawAction = {
       required: true,
       value: 'score',
       variables: false,
+      showOptionValue: false,
```

Comment:
nit: can update the description to be `Default is strength` instead? cus now it doesn't show the score value anymore